cam0:
  T_cam_imu:
  - [0.9983979874892823, 0.055340362109026, -0.011785707403144571, 0.0049436538123941234]
  - [-0.05400419704318103, 0.9941691990867925, 0.09333354321382291, -0.04449933201017387]
  - [0.016882099368026905, -0.09254754404503206, 0.9955651394118622, 0.091197153422931]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [1]
  camera_model: pinhole
  distortion_coeffs: [-0.0025558933226379255, -0.0007390597354944765, -0.0002914149200364255,
    -0.00048228924075012085]
  distortion_model: radtan
  intrinsics: [408.9142809820558, 404.81494926398256, 322.21895421246785, 242.59151285435885]
  resolution: [640, 480]
  rostopic: /acl_jackal/forward/infra1/image_rect_raw
  timeshift_cam_imu: 0.0
cam1:
  T_cam_imu:
  - [0.9983636781892014, 0.055593960729383586, -0.013389458650325264, -0.04438667476102461]
  - [-0.05410472657876852, 0.9941454558788465, 0.0935280231656661, -0.04460895887076704]
  - [0.01851066272086749, -0.09265054822212564, 0.9955266100309796, 0.09131149619108606]
  - [0.0, 0.0, 0.0, 1.0]
  T_cn_cnm1:
  - [0.9999986812462729, 0.00010428899715943761, -0.0016206879764819077, -0.04917787913320533]
  - [-0.00010397452754734297, 0.999999975753809, 0.00019411768634350685, -0.00012681690588648125]
  - [0.0016207081815252393, -0.0001939489200834284, 0.9999986678435145, 9.782143946612217e-05]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0]
  camera_model: pinhole
  distortion_coeffs: [-0.006195679031976135, 0.005706852393832609, -0.0005495427265571096,
    1.214244643822139e-05]
  distortion_model: radtan
  intrinsics: [408.97582288601444, 405.07065867820546, 322.91995279877983, 242.64964132363735]
  resolution: [640, 480]
  rostopic: /acl_jackal/forward/infra2/image_rect_raw
  timeshift_cam_imu: 0.0

T_imu_body:
  [1.0000, 0.0000, 0.0000, 0.0000,
  0.0000, 1.0000, 0.0000, 0.0000,
  0.0000, 0.0000, 1.0000, 0.0000,
  0.0000, 0.0000, 0.0000, 1.0000]
